### DISCUSSION TITLE
[WASM] VS2022 17.1 Preview 3 builds fail

### DISCUSSION THREAD

**Discussion by emceelovin** [unoplatform/uno#7872] [General]
Just a heads up, the following errors occur when building a WASM project. UWP works fine, but there also seems to be an issue with android deployment. Android builds succeed. Hope this helps! I'm not able to revert back to Preview 2 unfortunately, so I'll just sit back and wait since I still have Windows.Package at least. These are all analyzer errors it seems, so is it possible to build without analyzers like on the Android builds?

![image](https://user-images.githubusercontent.com/55720292/150283985-fc3d71bd-cbcb-42bc-b06d-1fc3915cdea8.png)

**Comment by jeromelaban** [comment]
Thanks for the report. You can try adding the following to your projects:

```xml
<PropertyGroup>
    <UnoUIUseRoslynSourceGenerators>false</UnoUIUseRoslynSourceGenerators>
</PropertyGroup>
```

It'll disable some parts of Roslyn used by uno.

**Reply by emceelovin** [reply]
No luck unfortunately, but I appreciate the help!

**Reply by emceelovin** [reply]
Something is causing something to feed dll's into the compiler instead of code files. Based on the error. Hopefully it's an "easy" fix.
